hack/tools/generate_versions.go

```diff
@@ -0,0 +1,212 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	mainBranch        = "master"
+	metricsServerRepo = "openshift/kubernetes-metrics-server"
+	versionFile       = "../../jsonnet/versions.yaml"
+	// This representation works for now.
+	minOCPVersion       = float32(4.10)
```

Comment:
In prometheus-operator, we use `github.com/blang/semver/v4` which is ok.

Reply:
`4.10` isn't really a semver... But I pushed a version using it by ignoring the `patch`